### PR TITLE
ionic RC1

### PR DESCRIPTION
**Pull request by WoH**
See: https://github.com/PokemonGoers/Catch-em-all/issues/90

**Big thanks to @johartl for a lot of his work on [Catch-em-all/tree/rc0](https://github.com/PokemonGoers/Catch-em-all/tree/rc0)**

### CODE REVIEW

**Comment by AlexanderLill** [comment]
Wow, that is ... a _lot_.

@sacdallago and @gyachdav should get their seminar statistics right - before you basically remove all our code ... ;)

Will review this tomorrow, wanna do this in one go before falling asleep...

**Comment by WoH** [comment]
I left everything in tact whenever possibe. I moved a lot of the stuff from /app... to /src, but I also had to scope the styles differently and as soon as that happens, github doesn't count it as moving a -> b but as removing a and adding b.

Also, when did statistics begin to matter? Just slip the build folder into one of your commits, add one to remove it and merge it before I can retaliate.

**Comment by MajorBreakfast** [comment]
First: Good work! But...



We're loosing scoped styles?

**Comment by sacdallago** [comment]
Hey guys will this PR fix the `ERROR NativeStorageError`? It's like super annoying!

I guess that's why the map is not loading? Maybe it also has to do with maxing out the requests at mapbox.

Just please merge if this is fine and fixes issues, we are like super late on schedule.

**Comment by WoH** [comment]
@sacdallago thanks for reminding me of that. However, this is not related to RC1 and this PR is already not the smallest. I opened another one that tackles that issue. Once that PR is merged and #136 is solved, the map will load and load without errors.

File: .travis.yml [hunk 1]
```diff
@@ -47,11 +46,11 @@ script:
 
   # Run frontend/app tests
   - cd ../ionic2
-  - npm test
+  # - npm test
```
Comment (AlexanderLill) [inline .travis.yml:49]: I guess this is removed because we do not have any tests?

**Review by AlexanderLill — CHANGES_REQUESTED** [review]
Looks good to me :)

**Review by AlexanderLill — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (WoH) [reply .travis.yml:49]: Exactly

**Comment by sacdallago** [comment]
Can this finally be merged?

**Comment by WoH** [comment]
Waiting for @johartl to confirm or add his thoughts.

**Comment by johartl** [comment]
I will have a look today. Sorry for the delay.

**Comment by WoH** [comment]
You need both I think, I was just about to fix that :D

**Comment by WoH** [comment]
@johartl Is this good to go? I re-enabled everything that I had to work around and I think this is ready now.

Edit: The agressive caching of the ServiceWorker should also massively decrease the data consumption.

**Comment by johartl** [comment]
Please give me some more time. Currently the Docker build is not working and I also want to test the Android build before merging it.

**Comment by WoH** [comment]
@sacdallago is coming after us, I agree we shouldn't make any overzealous moves though.

**Comment by sacdallago** [comment]
@WoH you bet I am

**Comment by johartl** [comment]
@sacdallago I'm sure the last thing you want is a broken Docker build though ;)
From my side this PR is good to go now!

**Review by johartl — APPROVED** [review]

**Comment by sacdallago** [comment]
@johartl oh, well, you know how to make a man happy, don't you?

**Comment by WoH** [comment]
**^**
KappaPride

I will merge as soon as I can conform that it integrates nicely with PM-1

**Comment by sacdallago** [comment]
oh wow, this happened

**Pull request merged**